_includes/footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="usa-grid">
+  <div class="usa-site-footer-content usa-width-one-half">
```

Comment:
Can we make the simple change so this includes `*-styleguide` in the class name? I'll also merge once conflicts are resolved.